ksql-common/src/main/java/io/confluent/ksql/util/KsqlConfig.java

```diff
@@ -202,6 +202,24 @@
   public static final String KSQL_AUTH_CACHE_MAX_ENTRIES_DOC = "Controls the size of the cache "
       + "to a maximum number of KSQL authorization responses entries.";
 
+  public static final String KSQL_INTERNAL_HIDDEN_TOPICS_CONFIG = "ksql.internal.hidden.topics";
```

Comment:
As above, I'd go with:

```suggestion
  public static final String KSQL_HIDDEN_TOPICS_CONFIG = "ksql.hidden.topics";
```